<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Tarjan</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threadNumSpinBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="threadNumLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>6</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of threads</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>50</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="View" name="drawPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="searchWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>250</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelSearch">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>220</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Concurrent Tarjan</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStatus">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="statusWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>50</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>110</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStatus_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>90</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStatus_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slow</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header location="global">view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
